packages/amplify_datastore/test/testData/Post.dart

```diff
@@ -0,0 +1,93 @@
+import 'package:amplify_datastore_plugin_interface/amplify_datastore_plugin_interface.dart';

```

Comment:
Not sure if we need license boilerplate for test files.

Reply:
I think we should. Added here.